raw_sql_profile.json

```diff
@@ -0,0 +1 @@
:[],"lineNumber":[],"columnNumber":[],"length":0},"nativeSymbols":{},"samples":{"stack":[],"time":[],"weight":[],"weightType":"samples","length":0},"stackTable":{"frame":[],"category":[],"subcategory":[],"prefix":[],"length":0},"resourceTable":{"length":0,"lib":[],"name":[],"host":[],"type":[]},"markers":{"data":[{"type":"GC_PAUSE","gc_by":"newobj","state":"sweeping"},{"type":"GC_PAUSE","gc_by":"newobj","state":"sweeping"},{"type":"GC_PAUSE","gc_by":"newobj","state":"sweeping"},{"type":"GC_PAUSE","gc_by":"newobj","state":"sweeping"},{"type":"GC_PAUSE","gc_by":"newobj","state":"sweeping"},{"type":"GC_PAUSE","gc_by":"newobj","state":"sweeping"},{"type":"GC_PAUSE","gc_by":"newobj","state":"sweeping"},{"type":"GC_PAUSE","gc_by":"newobj","state":"sweeping"},{"type":"GC_END_SWEEP"},{"type":"GC_PAUSE","gc_by":"newobj","state":"none"}],"name":[1,1,1,1,1,1,1,1,2,1],"startTime":[957833065.044,957833066.03,957833067.003,957833067.977,957833068.977,957833069.958,957833070.938,957833071.925,957833072.748,957833072.641],"endTime":[957833065.091,957833066.078,957833067.051,957833068.025,957833069.024,957833070.007,957833070.986,957833071.975,null,957833072.749],"phase":[1,1,1,1,1,1,1,1,0,1],"category":[6,6,6,6,6,6,6,6,6,6],"length":10},"stringArray":["<cfunc>","GC pause","GC end sweeping"]},{"name":"AR Pool Reaper","isMainThread":false,"processStartupTime":957833062.098,"registerTime":957833062.098,"pausedRanges":[],"pid":8807,"tid":8710896,"frameTable":{"address":[],"inlineDepth":[],"category":[],"subcategory":[],"func":[],"nativeSymbol":[],"innerWindowID":[],"implementation":[],"line":[],"column":[],"length":0},"funcTable":{"name":[],"isJS":[],"relevantForJS":[],"resource":[],"fileName":[],"lineNumber":[],"columnNumber":[],"length":0},"nativeSymbols":{},"samples":{"stack":[],"time":[],"weight":[],"weightType":"samples","length":0},"stackTable":{"frame":[],"category":[],"subcategory":[],"prefix":[],"length":0},"resourceTable":{"length":0,"lib":[],"name":[],"host":[],"type":[]},"markers":{"data":[{"type":"GC_PAUSE","gc_by":"newobj","state":"sweeping"},{"type":"GC_PAUSE","gc_by":"newobj","state":"sweeping"},{"type":"GC_PAUSE","gc_by":"newobj","state":"sweeping"},{"type":"GC_PAUSE","gc_by":"newobj","state":"sweeping"},{"type":"GC_PAUSE","gc_by":"newobj","state":"sweeping"},{"type":"GC_PAUSE","gc_by":"newobj","state":"sweeping"},{"type":"GC_PAUSE","gc_by":"newobj","state":"sweeping"},{"type":"GC_PAUSE","gc_by":"newobj","state":"sweeping"},{"type":"GC_END_SWEEP"},{"type":"GC_PAUSE","gc_by":"newobj","state":"none"}],"name":[1,1,1,1,1,1,1,1,2,1],"startTime":[957833065.044,957833066.03,957833067.003,957833067.977,957833068.977,957833069.958,957833070.938,957833071.925,957833072.748,957833072.641],"endTime":[957833065.091,957833066.078,957833067.051,957833068.025,957833069.024,957833070.007,957833070.986,957833071.975,null,957833072.749],"phase":[1,1,1,1,1,1,1,1,0,1],"category":[6,6,6,6,6,6,6,6,6,6],"length":10},"stringArray":["<cfunc>","GC pause","GC end sweeping"]}]}
```

Comment:
@byroot it looks like raw_sql_profile.json and insert_all_profile.json might have been committed unintentionally.

Reply:
Fixed in c17047bb6e8cbb7c455accce2b95a48af93b5c18

Reply:
Whoops. Thank you.